README.md

```diff
@@ -66,6 +67,25 @@ Then to install buildah follow the steps in this example:
   buildah --help
 ```
 
+In Ubuntu/Debian, you can use this command:
+
+```
+ add-apt-repository -y ppa:duggan/bats
+ apt-get -qq update
+ apt-get -qq install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libselinux1-dev
+```
+Then to install buildah on Ubuntu/Debian follow the steps in this example:
+
+```
+  mkdir ~/buildah
+  cd ~/buildah
+  export GOPATH=`pwd`
+  git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah
+  cd ./src/github.com/projectatomic/buildah
+  make install.tools install.libseccomp.sudo all runc validate TAGS="containers_image_ostree_stub apparmor seccomp"
```

Comment:
Some of this is stuff we do for Travis because it runs our tests on trusty.  Someone building on a more recent version (xenial or later) would probably be better off pulling in `ostree` packages from https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak.  Hopefully the packaged `libseccomp-dev` is new enough.

That first section probably wants to look more like 

```
  apt-get -qq update
  apt-get install software-properties-common
  add-apt-repository ppa:duggan/bats
  add-apt-repository ppa:alexlarsson/flatpak
  add-apt-repository ppa:gophers/archive
  apt-get install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev
  apt-get install golang-1.8
```

This line then becomes `PATH=/usr/lib/go-1.8/bin:$PATH make runc all TAGS="apparmor seccomp"`.